### PR TITLE
Finally it's possible to close the menu by pressing escape key

### PR DESCRIPTION
**Pull request by jbiron**
Sorry about the mess

### CODE REVIEW

**Comment by paullewis** [comment]
This looks good, just needs a rebase and we're gtg!

File: side-nav/side-nav.js [hunk 1]
```diff
@@ -47,6 +48,15 @@ class SideNav {
     document.addEventListener('touchstart', this.onTouchStart);
     document.addEventListener('touchmove', this.onTouchMove);
     document.addEventListener('touchend', this.onTouchEnd);
+    document.addEventListener('keydown', this.onKeyDown);
+  }
+  
+  onKeyDown (evt) {
+    if (evt.keyCode == 27 && this.sideNavEl.classList.contains('side-nav--visible')){
```
Comment (paullewis) [inline side-nav/side-nav.js]: Can you invert this:
```javascript
if (evt.keyCode !== 27)
  return;

if (this.sideNavEl.classList.contains('side-nav--visible'))
  return;

this.hideSideNav();
evt.preventDefault();
evt.stopPropagation();
```

**Comment by jbiron** [comment]
weird...

**Comment by paullewis** [comment]
Don't apologize, you going to open a new PR? It's a good addition!

**Comment by jbiron** [comment]
i'll try tonight, i've experience weird behaviour with the CLA

**Pull request closed without merging**